user/pages/03.commerce2/02.developer-guide/02.install-update/06.extending/docs.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Extending
+taxonomy:
+    category: docs
+---
+
+One of the primary strengths of both Commerce and Drupal is that they are open
+source projects. There is a large, active community of developers who
+contribute to these and related projects. As a result, you will find that it's
+easy to extend Commerce with freely available, "contributed" modules. And if
+you can't find something to meet your needs, you can always create your own
+modules as well. Contributed modules as well as extensive documentation about
+extending your Drupal site can be found on [drupal.org]. A good place to start
+is [Extending Drupal 8].
+
+For Commerce, you can find everything from modules that provide major types of
+functionality to smaller modules that provide unique, specialized functionality.
+For example:
+
+* [Commerce Shipping] makes it possible to ship physical products to your customers.
+* [Commerce Recurring] makes it possible to provide subscriptions and recurring billing.
+* [Commerce Variation Cart Form] provides an alternative add-to-cart form for specific use cases.
```

Comment:
Let's not recommend Commerce Variation Cart Form. We'll be deprecating it with Commerce functionality in a future release, and our examples should be more major. Two examples are enough, if not, we can point to Product Bundle or Price list.